tests/unit/algorithms/detection/test_helpers.py

```diff
@@ -33,7 +33,7 @@
     "src/otx/algorithms/detection/configs/instance_segmentation", "efficientnetb2b_maskrcnn"
 )
 DEFAULT_DET_RECIPE_CONFIG_PATH = "src/otx/recipes/stages/detection/incremental.py"
-DEFAULT_ISEG_RECIPE_CONFIG_PATH = "src/otx/recipes/stages/instance-segmentation/incremental.py"
+DEFAULT_ISEG_RECIPE_CONFIG_PATH = "src/otx/recipes/stages/instance_segmentation/incremental.py"
```

Comment:
These path changes are due to documentation warnings? Interesting.

Reply:
yes. sphinx tried to find `otx.recipes.stages.instance` module when the path name includes `-`.